plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/interactive/layered/ConstraintsLanguageClient.xtend

```diff
@@ -0,0 +1,25 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ * 
+ * Copyright 2019 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ * 
+ * This code is provided under the terms of the Eclipse Public License (EPL).
+ */
+package de.cau.cs.kieler.klighd.lsp.interactive.layered
+
+import org.eclipse.lsp4j.jsonrpc.services.JsonSegment
+import org.eclipse.lsp4j.services.LanguageClient
+
+/**
+ * @author jet, cos
+ *
+ */
+ @JsonSegment("keith/constraintsLC")
+interface ConstraintsLanguageClient extends LanguageClient {
+    
+}
```

Comment:
Empty interface. What is this for?

Reply:
To add server client notifications in the future. Do you suggest to remove it since it is not needed at this time?

Reply:
Remove it as it is a non necessary class.